docs/data/data-grid/overview/overview.md

```diff
@@ -7,119 +7,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/grid/
+{{"demo": "DataGridDemo.js", "defaultCodeOpen": false, "bg": "inline"}}
 
-### Pro plan [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+## Pro version [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
 
-The Pro plan extends the features available in the Community version to support more complex use cases.
-It adds features like advanced filtering, column pinning, column and row reordering, support for tree data, and virtualization to handle bigger datasets.
+The Pro plan expands on the Community version to support more complex use cases with features like advanced filtering, column pinning, column and row reordering, support for tree data, and virtualization to handle larger datasets.
 
-The demo below displays 31 columns and 100,000 rows—over 3 million cells in total.
+Pro features are denoted by the blue cube icon (<span class="plan-pro"></span>) throughout the documentation.
 
-```js
-import { DataGridPro } from '@mui/x-data-grid-pro';
-```
+The demo below displays 31 columns and 100,000 rows—over three million cells in total:
 
 {{"demo": "DataGridProDemo.js", "defaultCodeOpen": false, "disableAd": true, "bg": "inline"}}
 
-### Premium plan [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+## Premium version [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
 
-The Premium plan includes everything from Pro, as well as advanced features for data analysis and large dataset management, like row grouping with aggregation functions (such as sum and average) and the ability to export to Excel files.
+The Premium plan includes all Pro features as well as more advanced features for data analysis and large dataset management, such as row grouping with aggregation functions (like sum and average) and the ability to export to Excel files.
 
-Visit the [showcase page](/x/react-data-grid/demo/) for a comprehensive overview of all features exclusive to this plan.
+Premium features are denoted by the golden cube icon (<span class="plan-premium"></span>) throughout the documentation.
 
 The demo below groups rows by commodity name, and uses an aggregation function to calculate the sum of quantities for each group and in total (displayed in a summary row).
-You can experiment with grouping other columns in the columns' header menus.
-
-And since you're here, try exporting to Excel and copying and pasting data from-to Excel tables.
-
-```js
-import { DataGridPremium } from '@mui/x-data-grid-premium';
-```
+You can experiment with grouping other columns in the column header menus.
+You can also try exporting to Excel, and copying and pasting data to and from Excel tables.
 
 {{"demo": "DataGridPremiumDemo.js", "defaultCodeOpen": false, "disableAd": true, "bg": "inline"}}
 
-### MIT vs. commercial
-
-Please see [the Licensing page](/x/introduction/licensing/) for details.
-
-## Features
```

Comment:
Unless we have something more sophisticated like https://deploy-preview-15757--material-ui-x.netlify.app/x/react-date-pickers/, it looks like we need to restore this list, no? How do we keep visitors to continue considering the component without overwhelming them with docs info otherwise?

Reply:
Before this PR there were 3 different feature lists: on the Overview, Getting Started, and Demo pages. I really don't think there should be more than 1 list. I renamed Demo to Feature Showcase to try to make it the single source of truth for features.